website/src/content/main-concepts/api-gateway.mdx

```diff
@@ -10,9 +10,52 @@ To solve these issues, the Merchant Center provides an **API Gateway**, which pr
 
```

Comment:
To let the index navigation better represent the actual document structure I suggest to give the big image a heading.  You can pick something else than what I propse here, but a heading would be good. 

```suggestion

# Architecture Overview

```